nuxeo-common/src/main/java/org/nuxeo/common/utils/ZipUtils.java

```diff
@@ -469,7 +407,7 @@ public static InputStream getEntryContentAsStream(File file, String entryName) t
     public static String getEntryContentAsString(File file, String entryName) throws IOException {
         InputStream resultStream = getEntryContentAsStream(file, entryName);
         // resultStream has been closed by FileUtils call
-        return FileUtils.read(resultStream);
+        return IOUtils.toString(resultStream, Charsets.UTF_8);
```

Comment:
Sure, I wanted to do a second pass on ZipUtils because a lot of method are not used anymore and none are marked deprecated...